### DISCUSSION TITLE
ENABLING DAC CONTROL & USING NOISE/WAVE TOGGLE

### DISCUSSION THREAD

**Discussion by jerold49** [2xAA/genmdm-editor#110] [Q&A] [answered]
Both my online & stand alone genmdm-editors don't response to either the DAC CONTROL & Noise/Wave toggle as seen in your video. (Note creating a wave in the box as you demo'd only gives me half a wave plus I hear no differences.) I'm using the latest web based editor version on Chrome & latest downloaded version on a Mac. Any suggestions? Thanks.

**Comment by 2xAA** [comment, marked as answer]
Hi @jerold49,

Please make sure you're using channel 6.
The DAC Control feature only works on that channel.

Thanks,
Sam
